Source/FakeItEasy.Specs/LazySpecs.cs

```diff
@@ -50,5 +70,12 @@ protected override IFoo Create()
                 return instance;
             }
         }
+
+        public class Bar
+        {
+            internal Bar()

```

Comment:
In this case, I can make it private. But in TaskSpecs, it has to be internal, because I'm explicitly calling it in one of the scenarios. However I'm not sure the scenario with a configured method is really useful, I added it only for consistency with the existing specs. It's already covered extensively in other specs and tests.